src/main/java/cat/udl/eps/softarch/demo/domain/Offer.java

```diff
@@ -1,29 +1,20 @@
 package cat.udl.eps.softarch.demo.domain;
 
 import com.fasterxml.jackson.annotation.JsonIdentityReference;
-
 import javax.persistence.*;
-import javax.validation.constraints.NotBlank;
-
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-
-import java.math.BigDecimal;
-import java.time.ZonedDateTime;
+import java.util.Date;
 
 @Entity
 @Data
-@EqualsAndHashCode(callSuper = false)
-public class Offer extends UriEntity<Long> {
-    @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
-    private Long id;
+@EqualsAndHashCode(callSuper = true)
+public class Offer extends Announcement {
 
-    private ZonedDateTime dateTime;
+    private Date dateTime;
```

Comment:
Better use ZonedDateTime to avoid problems later when showing dates through the user frontend

Reply:
For an example, you can check: https://github.com/UdL-EPS-SoftArch/mycircular-api/pull/14/commits/6fbe7f8d555cd70f1617bb4daf6b0cdd4f83d362